### PR TITLE
renamed Nimiq 1.0 to PoW

### PR DESCRIPTION
**Pull request by onmax**

### CODE REVIEW

**Comment by onmax** [comment]
@ines-toupeira I know you are in TO, this is not urgent :)

but since we need someone to track this kind of changes, I am making sure you are in the loop. If there is a problem with this change, please let me know.

File: build/nimiq-1.0/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 search: false
 ---
-# Nimiq 1.0
+# Nimiq PoW
 
-This is a port of the old content of the Nimiq 1.0.
+This is a port of the old content of the Nimiq PoW.
```
Comment (sisou) [inline build/nimiq-1.0/README.md]: The Nimiq PoW _what_? There's a word missing at the end of the sentence. _Documentation_? _Build instructions_?

File: build/nimiq-1.0/get-started.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 search: false
 ---
 
-# Get started with Nimiq 1.0
+# Get started with PoW
```
Comment (sisou) [inline build/nimiq-1.0/get-started.md]:
```suggestion
# Get started with Nimiq PoW
```

File: build/nimiq-1.0/README.md (hunk 1 above)
Comment (onmax) [reply build/nimiq-1.0/README.md]: It is just a README.md. It does not get shown in the website: https://www.nimiq.com/developers/build/nimiq-1.0/

**Review by ines-toupeira — APPROVED** [review]

**Pull request merged**